docs/documentation/quartz-3.x/tutorial/overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Library Overview
+---
+
+# The Quartz API
+
+The key interfaces and classes of the Quartz API are:
+
+
+| Type | |
+|-|--|
+| `IScheduler` | the main API for interacting with the scheduler |
+| `IJob` | an interface to be implemented by components that you wish to have executed by the scheduler |
+| `IJobDetail` | used to define instances of Jobs |
+| `ITrigger` | a component that defines the schedule upon which a given Job will be executed, job can have multiple associated triggers |
+| `JobBuilder` | used to define/build JobDetail instances, which define instances of Jobs |
+| `TriggerBuilder` | used to define/build Trigger instances |
+| `SchedulerBuilder` | used to define/build scheduler instances, requires Quartz 3.1 or later |
+
+In this tutorial for readability's sake following terms are used interchangeably: `IScheduler` and `Scheduler`, `IJob` and `Job`, `IJobDetail` and `JobDetail`, `ITrigger` and `Trigger`.
```

Comment:
great clarification!

Reply:
That was actually already there, but I do think its helpful too!